README.md

```diff
@@ -12,6 +12,9 @@ Please feel free to open a PR against `servers.json` in this repository, and we'
 
 The `addresses` array in each server object is actually an array of IP _suffixes_. For example, `"addresses": ["lunar.gg", "other.domain"]` will match `lunar.gg`, `na.lunar.gg`, `play.other.domain`, and so on.
 
+## Primary Info & Minecraft Versions
+We also require your server's primary connection information and allowed client versions. The primary address must not be a subdomain, and should also be included in the `addresses` array. The `minecraftVersions` field must be an array of client versions allowed on your Minecraft server. *(ie. 1.18, 1.19)*; The `primaryMinecraftVersion` field must be a version included in the `minecraftVersions` array.
```

Comment:
The primary address can be a subdomain, probably should just remove that portion of the sentence.